README.md

```diff
@@ -87,7 +88,7 @@ Runs:
 * __js(gulp, config)__ Config accepts:
 	- js: `String` Path to your main JavaScript file. (Default: './main.js' and checks your bower.json to see if it's in its main key)
 	- buildJs: `String` Name of the built JavaScript bundle. (Default: 'main.js')
-	- buildFolder: `String` Path to directory where the built file will be created. (Default: './build/')
+	- buildFolder: `String` Path to directory where the built file will be created. If set to `null`, files won't be saved. (Default: './build/')
```

Comment:
I don't mind calling it a string or passing it null. I chose null preventing the extreme edge case of having a directory called disabled. Why is it not a good idea to give null some context?